Build fails on 32 bit due to uint32/long int format char issue

## Description

Describe the problem here.
Build fails on 32 bit due to uint32/long int format char issue

## Steps to Reproduce

1. Compile the source on Fedora 25 32 bit system 

## Expected Behavior
The source compiles.
## Actual Behavior
The compilation fails with: 

[  131s] libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../src/rnpv -I../../include -Werror -Wall -Wpointer-arith -I/usr/include/botan-2 -I/usr/include/json-c -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m32 -march=i686 -mtune=atom -fasynchronous-unwind-tables -std=c11 -D_GNU_SOURCE -I/usr/include/botan-2 -I/usr/include/botan-2 -I/usr/include/botan-2 -I/usr/include/botan-2 -I/usr/include/botan-2 -I/usr/include/botan-2 -I/usr/include/botan-2 -I/usr/include/botan-2 -c key_store_kbx.c  -fPIC -DPIC -o .libs/librnp_la-key_store_kbx.o
[  131s] key_store_kbx.c: In function 'rnp_key_store_kbx_parse_pgp_blob':
[  131s] key_store_kbx.c:201:83: error: format '%lu' expects argument of type 'long unsigned int', but argument 4 has type 'uint32_t {aka unsigned int}' [-Werror=format=]
[  131s]                  "Serial number is %u and it's bigger that blob size it can use: %lu\n",
[  131s]                                                                                    ^
[  131s] cc1: all warnings being treated as errors
[  131s] Makefile:620: recipe for target 'librnp_la-key_store_kbx.lo' failed


PR will follow soon.

--- a/src/lib/key_store_kbx.c
+++ b/src/lib/key_store_kbx.c
@@ -198,9 +198,9 @@ rnp_key_store_kbx_parse_pgp_blob(kbx_pgp_blob_t *pgp_blob)
 
     if (pgp_blob->sn_size > pgp_blob->blob.length - (image - pgp_blob->blob.image)) {
         fprintf(stderr,
-                "Serial number is %u and it's bigger that blob size it can use: %lu\n",
+                "Serial number is %u and it's bigger that blob size it can use: %u\n",
                 pgp_blob->sn_size,
-                pgp_blob->blob.length - (image - pgp_blob->blob.image));
+                (uint32_t)(pgp_blob->blob.length - (image - pgp_blob->blob.image)));
         return 0;
     }
 

